docs/samples/product_recommendation_demo.ipynb

```diff
@@ -4,38 +4,40 @@
       "cell_type": "markdown",
       "metadata": {},
       "source": [
-        "# Demo Notebook: Feathr Feature Store on Azure\n",
+        "# Product Recommendation with Feathr\n",
         "\n",
         "This notebook demonstrates how Feathr Feature Store can simplify and empower your model training and inference. You will learn:\n",
         "\n",
         "1. Define sharable features using Feathr API\n",
         "2. Create a training dataset via point-in-time feature join with Feathr API\n",
-        "3. Materialize features to online store and then retrieve them with Feathr API"
+        "3. Materialize features to online store and then retrieve them with Feathr API\n",
```

Comment:
`4. register feature with register API` could be added according to the notebook content. 

And I would like to propose moving `register feature step` to step 2. Users are usually confused and think the feature registry are related to spark environment.

Reply:
Great callout, addressed